account_invoice_export/views/transmit_method.xml

```diff
@@ -14,6 +14,16 @@
                     attrs="{'invisible': [('customer_ok', '=', False)]}"
                 >
                     <field name="send_through_http" />
+                    <div
+                        role="alert"
+                        class="alert alert-info"
+                        colspan="2"
+                        attrs="{'invisible': [('send_through_http', '=', False)]}"
+                    >
+                      <span
+                        >By default the PDF of the invoice will be send using the connection parameters below (basic authenticaiton).<br
```

Comment:
```suggestion
                        >By default the PDF of the invoice will be sent using the connection parameters below (basic authenticaiton).<br
```
also, I'd remove the BR and use 2 P so that you won't have any markup in the translation

Reply:
fixed